README.md

```diff
@@ -27,20 +27,26 @@ We use GitHub Issues to organize our work.  Here's some quick reading about [how
 
 
 ## Environment Setup
-If you're just starting out with github, they have a [guide/tutorial](https://help.github.com/articles/fork-a-repo/) about forking and cloning.
-
-1. Fork this repo.
-2. Navigate to the directory where you're going to keep this code.
-3. `git clone` using the forked repo's address.
-4. Navigate into the new repository's directory with `cd cliff-effects` (on OSX you can drag and drop the folder into the terminal window).
-5. Make sure you're on the main branch with `git checkout dev`.  *Do not touch `master`.*
-6. Check your node.js version in your terminal with `node --version`. Versions >=5.0 are preferred. If you don't get a version number (it would look something like 'v6.11.1'), [install node](https://nodejs.org/en/).
-7. Install all necessary node packages with `npm install`. This might take a few minutes.
-8. Now you can try out the current UI or play with the code!
-9. To run a local version of the site so you can play around or debug your changes, type `npm start`. It will automatically open a browser window and run the code locally. If you make changes to a file and save it, the page will automatically update and use the new code. Would also recommend getting the [React Developer Tools](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=4&cad=rja&uact=8&ved=0ahUKEwiZ__6Vg_jVAhWQ14MKHczrDtoQFgg4MAM&url=https%3A%2F%2Fchrome.google.com%2Fwebstore%2Fdetail%2Freact-developer-tools%2Ffmkadmapgofadopljbjfkapdkoienihi%3Fhl%3Den&usg=AFQjCNEv0udXgBoaukzJa59I_vufhScUbQ) extension for Chrome.
-10. `Ctrl-C` in the terminal to stop the app when it's running.
-
-To keep up to date with the latest changes to the `dev` branch, you'll need to use git to add a new remote branch with this repo's address. The page about [git workflow](https://github.com/codeforboston/cliff-effects/wiki/Git-Workflow) includes some notes about that.
+To follow these instructions, it'd help if you:
+- have mentally perpared for experimentation and learning and/or
+- know how to open your command prompt/terminal
+- know a bit about how to navigate to different folders in your command prompt/terminal
+- know a bit about installing software for your command prompt/terminal, like nodejs
+
+Instructions:
+1. If you know how, fork and clone the repo, and create a link (a remote branch) to our repo. If not, see this [github guide](https://help.github.com/articles/fork-a-repo/). Go through all the steps.
```

Comment:
Suggested change:
> Fork and clone the repo, creating a link (remote branch) to the Code for Boston repo.  [How to: fork a repo.]((https://help.github.com/articles/fork-a-repo/)

Keeps things succinct by omitting qualifiers.

Reply:
Nice combination, but the feedback I got was that the way the instruction was presented was overwhelming because at the start it seemed to assume you knew how to do those actions. We get a lot of people who are new to these technologies. Perhaps something like:
> 1. If you're new to github look at [How to: fork a repo](...). Otherwise, fork and clone the repo, creating a link (remote branch) to the Code for Boston repo.

A little shorter, but reassures new users. Does that work?

Reply:
Yep, sounds good